modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb

```diff
@@ -115,7 +114,8 @@ def flashes
 
         def shared_validation
           validate_json_schema
-          validate_form_526_submission_values!(target_veteran)
+          error = validate_form_526_submission_values!(target_veteran)
```

Comment:
does this make sense to be a `!` method? seems like ! implies that the methods `modify the object it's called on`. We're verifying data here, not manipulating / modifying it.

Reply:
@FonzMP  that is a good question. I think originally, since these would throw exceptions, or could, that made sense. 
 But now, since they do not raise errors anymore the methods inside the file no longer all need the bang on them.  I think it would make sense to remove all the `!` at this point.

Reply:
I created a clean up ticket to address the bangs on the methods.  All in all it will effect about 162 lines of code to bring those all in line so thought that might be better off in it's own clean up ticket.